lib/controllers/frontend/spree/users_controller.rb

```diff
@@ -28,14 +28,13 @@ def create
   def update
     if @user.update(user_params)
       spree_current_user.reload
+      redirect_url = spree.account_url
 
       if params[:user][:password].present?
         # this logic needed b/c devise wants to log us out after password changes
-        unless Spree::Auth::Config[:signout_after_password_change]
-          bypass_sign_in(@user)
-        end
+        Spree::Auth::Config[:signout_after_password_change] ? redirect_url = spree.login_url : bypass_sign_in(@user)
```

Comment:
I find this inline assignment in a ternary operator quite hard to read. 😛 Could you just turn this into a regular if condition?